src/api/operations.ts

```diff
@@ -272,22 +275,25 @@ export const drawEllipse = (options: {
       y: options.y,
       xScale: options.xScale,
       yScale: options.yScale,
+      rotate: options.rotate,
     }),
 
     // prettier-ignore
     options.color && options.borderWidth ? fillAndStroke()
-  : options.color                      ? fill()
-  : options.borderColor                ? stroke()
+  : options.color ? fill()
+  : options.borderColor ? stroke()
   : closePath(),
 
     popGraphicsState(),
   ].filter(Boolean) as PDFOperator[];
 
+
 export const drawSvgPath = (
   path: string,
   options: {
     x: number | PDFNumber;
     y: number | PDFNumber;
+    rotate: Rotation;
```

Comment:
This also must be marked as optional:
```suggestion
    rotate?: Rotation;
```